resources/js/Pages/Tasks/Redesign/tasks-redesign.css

```
@@ -0,0 +1,278 @@
+    background: #f8f9fb;
+}
+
+/* Board cards must never widen their column — long headings/descriptions are
+   clamped rather than pushing the board into horizontal overflow. */
+.tasks-clamp-2,
+.tasks-clamp-3 {
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)